tools/ci_build/github/azure-pipelines/linux-gpu-ci-pipeline.yml

```diff
@@ -183,7 +183,7 @@ stages:
       skipComponentGovernanceDetection: true
     workspace:
       clean: all
-    pool: onnxruntime-Linux-GPU-A10
+    pool: onnxruntime-Linux-GPU-T4
```

Comment:
Just curious - Is the instability related to specific set of tests or general VM instabilities ?

Reply:
No.  It's a workaround to fix the random error.
"CUDA failure 100: no CUDA-capable device is detected"
So far, we find no clue to solve it.
https://github.com/microsoft/onnxruntime/pull/19569